devops/local_conf/nginx.conf

```diff
@@ -13,18 +13,20 @@ http {
     server {
         include /etc/nginx/mime.types;
         client_max_body_size 10g;
+
         location / {
-             add_header "Access-Control-Allow-Origin"  *;
+            add_header 'Access-Control-Allow-Origin' '*' always;
 
             # Preflighted requests
             if ($request_method = OPTIONS ) {
-                add_header "Access-Control-Allow-Origin"  *;
-                add_header "Access-Control-Allow-Methods" "GET, POST, OPTIONS, HEAD, PUT, DELETE";
-                add_header "Access-Control-Allow-Headers" "*";
-                add_header 'Access-Control-Max-Age' 1728000;
-                add_header 'Content-Length' 0;
+                add_header 'Access-Control-Allow-Origin' '*' always;
+                add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS, HEAD, PUT, DELETE' always;
+                add_header 'Access-Control-Allow-Headers' '*' always;
+                add_header 'Access-Control-Max-Age' 1728000 always;
```

Comment:
Same question as above for `geoapi-services/nginx.conf`: Does this actually override the browser limits enforced in FF and Chromium?

Reply:
fixed in e4bfad207150aaafcb0f875b914b3b5cd885d8f0